tests/unit/test_finder.py

```diff
@@ -701,3 +701,44 @@ def test_link_package_versions_substring_fails(self):
         link = Link('http:/yo/pytest_xdist-1.0-py2.py3-none-any.whl')
         result = self.finder._link_package_versions(link, self.search_name)
         assert result == [], result
+
+
+def test_get_indexes_location():
+    """Check that the canonical name is on all indexes"""
+    finder = PackageFinder(
+        [], ['file://index1/', 'file://index2'], session=PipSession())
+    locations = finder._get_indexes_locations(
```

Comment:
I see above that there's a test that gets the result of `self.finder._link_package_versions` but I think we should be making these public if we're going to test them.